services/gundeck/src/Gundeck/Push.hs

```diff
@@ -180,7 +178,7 @@ splitPush clientsFull p = do
               Set.filter (\c -> c.clientId `elem` toList cs) allClients
             RecipientClientsAll -> allClients
             RecipientClientsTemporaryOnly -> mempty
-          isClientForRabbitMq c = ClientSupportsConsumableNotifications `Set.member` c.clientCapabilities.fromClientCapabilityList
+          isClientForRabbitMq c = supportsConsumableNotifications c
```

Comment:
```suggestion
          isClientForRabbitMq = supportsConsumableNotifications
```
Can't we eta-reduce here?

Reply:
We can probably even get rid of the helper :smile:

Reply:
Beta reduction seems to be fitting as well :upside_down_face: